### ISSUE TITLE
WebRTC dont work with core-2021.12.0b0

### ISSUE THREAD

**Issue by Kenneth198** [AlexxIT/WebRTC#182]
Homeassistant core-2021.12.0b4. WEbRTC not load.

**Comment by zvldz** [comment]
Very informative ))

**Comment by aeros2** [comment]
I have such a problem. HA Supervisor 2021.11.5 I install the master version via HACS.

`2021-12-09 14:33:02 ERROR (MainThread) [homeassistant.setup] Error during setup of component webrtc
Traceback (most recent call last):
File "/usr/src/homeassistant/homeassistant/setup.py", line 254, in _async_setup_component
result = await task
File "/config/custom_components/webrtc/__init__.py", line 82, in async_setup
await utils.init_resource(hass, url_path, str(version))
File "/config/custom_components/webrtc/utils.py", line 93, in init_resource
if not item['url'].startswith(url):
KeyError: 'url'`

**Comment by zvldz** [comment]
>I have such a problem. HA Supervisor 2021.11.5 I install the master version via HACS.

I just updated webrtc to the master version on HA 2021.11.5, I have no such errors ((

**Comment by aeros2** [comment]
> > У меня такая проблема. HA Supervisor 2021.11.5 Устанавливаю мастер-версию через HACS.
> 
> Я только что обновил webrtc до мастер-версии на HA 2021.11.5, таких ошибок у меня нет ((
> 
> Попробуйте переустановить компонент

I tried reinstalling the master version many times. Manually is also a problem.

**Comment by zvldz** [comment]
>I tried reinstalling the master version many times. Manually is also a problem.

Did it work for you earlier or is this the first attempt?

**Comment by aeros2** [comment]
> > Я много раз пытался переустановить основную версию. Вручную - тоже проблема.
> 
> У вас это сработало раньше или это первая попытка?

the first time there was an error 2-3 days ago, it was not there before.

**Comment by aeros2** [comment]
Success!
A WebRTC Camera configuration record has been created

Integration failed to load, try restarting Home Assistant.

**Comment by Arquiteto** [comment]
I can confirm there is an error in new version. I am on 2021.12.0b5 and have an error as follows:

```
Logger: homeassistant.setup
Source: custom_components/webrtc/utils.py:74
Integration: webrtc (documentation, issues)
First occurred: 15:45:30 (1 occurrences)
Last logged: 15:45:30

Error during setup of component webrtc
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/setup.py", line 229, in _async_setup_component
    result = await task
  File "/config/custom_components/webrtc/__init__.py", line 75, in async_setup
    utils.register_static_path(hass.http.app, url_path, path)
  File "/config/custom_components/webrtc/utils.py", line 74, in register_static_path
    app['allow_cors'](route)
  File "/usr/local/lib/python3.9/site-packages/aiohttp/web_app.py", line 186, in __getitem__
    return self._state[key]
KeyError: 'allow_cors'

```

**Comment by jine** [comment]
> I can confirm there is an error in new version. I am on 2021.12.0b5 and have an error as follows:
> 
> ```
> Logger: homeassistant.setup
> Source: custom_components/webrtc/utils.py:74
> Integration: webrtc (documentation, issues)
> First occurred: 15:45:30 (1 occurrences)
> Last logged: 15:45:30
> 
> Error during setup of component webrtc
> Traceback (most recent call last):
>   File "/usr/src/homeassistant/homeassistant/setup.py", line 229, in _async_setup_component
>     result = await task
>   File "/config/custom_components/webrtc/__init__.py", line 75, in async_setup
>     utils.register_static_path(hass.http.app, url_path, path)
>   File "/config/custom_components/webrtc/utils.py", line 74, in register_static_path
>     app['allow_cors'](route)
>   File "/usr/local/lib/python3.9/site-packages/aiohttp/web_app.py", line 186, in __getitem__
>     return self._state[key]
> KeyError: 'allow_cors'
> ```

https://github.com/AlexxIT/WebRTC/pull/177/files#diff-268266fb86d71bd71ed88a54850bd9ff45a46f014acb42741ed28173a817cb8eR74

**Comment by dale3h** [comment]
> > I can confirm there is an error in new version. I am on 2021.12.0b5 and have an error as follows:
> > ```
> > Logger: homeassistant.setup
> > Source: custom_components/webrtc/utils.py:74
> > Integration: webrtc (documentation, issues)
> > First occurred: 15:45:30 (1 occurrences)
> > Last logged: 15:45:30
> > 
> > Error during setup of component webrtc
> > Traceback (most recent call last):
> >   File "/usr/src/homeassistant/homeassistant/setup.py", line 229, in _async_setup_component
> >     result = await task
> >   File "/config/custom_components/webrtc/__init__.py", line 75, in async_setup
> >     utils.register_static_path(hass.http.app, url_path, path)
> >   File "/config/custom_components/webrtc/utils.py", line 74, in register_static_path
> >     app['allow_cors'](route)
> >   File "/usr/local/lib/python3.9/site-packages/aiohttp/web_app.py", line 186, in __getitem__
> >     return self._state[key]
> > KeyError: 'allow_cors'
> > ```
> 
> https://github.com/AlexxIT/WebRTC/pull/177/files#diff-268266fb86d71bd71ed88a54850bd9ff45a46f014acb42741ed28173a817cb8eR74

Thank you for linking this. Between this and https://github.com/blakeblackshear/frigate-hass-integration/issues/181 I was able to get Frigate and WebRTC back up and running on core 2021.12.0b5.

**Comment by chises** [comment]
I can confirm that this fixed the issue with core-2021.12.0b6, too.
https://github.com/AlexxIT/WebRTC/pull/177/files#diff-268266fb86d71bd71ed88a54850bd9ff45a46f014acb42741ed28173a817cb8eR74

**Comment by AlexxIT** [comment]
Duplicate #156 and already fixed in master version
